### PR TITLE
tidb-lightning: tidb.pd-addr supports multiple addresses

### PR DESCRIPTION
**Pull request by hfxsd**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:https://github.com/pingcap/docs-cn/pull/16185
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: dm/task-configuration-file-full.md [hunk 1]
```diff
@@ -172,8 +172,8 @@ loaders:
     # range-concurrency: 16
     # Only available for physical import mode. Whether to enable compression when sending KV data to TiKV. Currently, only Gzip compression is supported and can be specified using either "gzip" or "gz". Compression is not enabled by default.
     # compress-kv-pairs: ""
-    # One or more PD server addresses. If no address is specified, use the PD address information from the TiDB query by default.
-    # pd-addr: "192.168.0.1:2379"
+    # One or more PD server addresses. If no address is specified, use the PD address information from the TiDB query by default. Starting from v7.6.0, TiDB supports setting multiple PD addresses.
+    # pd-addr = "172.16.31.4:2379,56.78.90.12:3456"
```
Comment (hfxsd) [inline dm/task-configuration-file-full.md:176]:
```suggestion
    # One or more PD server addresses. If no address is specified, use the PD address information from the TiDB query by default.
    # pd-addr: "192.168.0.1:2379"
```

File: tidb-lightning/deploy-tidb-lightning.md [hunk 2]
```diff
@@ -108,8 +108,8 @@ Refer to the [Download TiDB Tools](/download-ecosystem-tools.md) document to dow
     password = ""
     # Table schema information is fetched from TiDB via this status-port.
     status-port = 10080
-    # An address of pd-server.
-    pd-addr = "172.16.31.4:2379"
+    # An address of pd-server.Starting from v7.6.0, TiDB supports setting multiple PD addresses.
```
Comment (hfxsd) [inline tidb-lightning/deploy-tidb-lightning.md:111]:
```suggestion
    # An address of pd-server. Starting from v7.6.0, TiDB supports setting multiple PD addresses.
```

File: tidb-lightning/deploy-tidb-lightning.md [hunk 3]
```diff
@@ -108,8 +108,8 @@ Refer to the [Download TiDB Tools](/download-ecosystem-tools.md) document to dow
     password = ""
     # Table schema information is fetched from TiDB via this status-port.
     status-port = 10080
-    # An address of pd-server.
-    pd-addr = "172.16.31.4:2379"
+    # An address of pd-server. Starting from v7.6.0, TiDB supports setting multiple PD addresses.
+    pd-addr = "172.16.31.4:2379,56.78.90.12:3456"
```
Comment (hfxsd) [inline tidb-lightning/deploy-tidb-lightning.md:112]:
```suggestion
    # An address of pd-server.
    pd-addr = "172.16.31.4:2379"
```

File: tidb-lightning/tidb-lightning-command-line-full.md [hunk 4]
```diff
@@ -23,7 +23,7 @@ You can configure the following parameters using `tidb-lightning`:
 | `--backend <backend>` | Select an import mode. `local` refers to [physical import mode](/tidb-lightning/tidb-lightning-physical-import-mode.md); `tidb` refers to [logical import mode](/tidb-lightning/tidb-lightning-logical-import-mode.md). | `tikv-importer.backend` |
 | `--log-file <file>` | Log file path. By default, it is `/tmp/lightning.log.{timestamp}`. If set to '-', it means that the log files will be output to stdout. | `lightning.log-file` |
 | `--status-addr <ip:port>` | Listening address of the TiDB Lightning server | `lightning.status-port` |
-| `--pd-urls <host:port>` | PD endpoint address | `tidb.pd-addr` |
+| `--pd-urls <host1:port1,host2:port2)>` | PD endpoint address. Starting from v7.6.0, TiDB supports setting multiple PD addresses. | `tidb.pd-addr` |
```
Comment (lance6716) [inline tidb-lightning/tidb-lightning-command-line-full.md:26]:
```suggestion
| `--pd-urls <host1:port1,host2:port2>` | PD endpoint address. Starting from v7.6.0, TiDB supports setting multiple PD addresses. | `tidb.pd-addr` |
```

**Review by lance6716 — APPROVED** [review]
rest lgtm

File: tidb-lightning/tidb-lightning-command-line-full.md [hunk 5]
```diff
@@ -23,7 +23,7 @@ You can configure the following parameters using `tidb-lightning`:
 | `--backend <backend>` | Select an import mode. `local` refers to [physical import mode](/tidb-lightning/tidb-lightning-physical-import-mode.md); `tidb` refers to [logical import mode](/tidb-lightning/tidb-lightning-logical-import-mode.md). | `tikv-importer.backend` |
 | `--log-file <file>` | Log file path. By default, it is `/tmp/lightning.log.{timestamp}`. If set to '-', it means that the log files will be output to stdout. | `lightning.log-file` |
 | `--status-addr <ip:port>` | Listening address of the TiDB Lightning server | `lightning.status-port` |
-| `--pd-urls <host:port>` | PD endpoint address | `tidb.pd-addr` |
+| `--pd-urls <host1:port1,host2:port2>` | PD endpoint address. Starting from v7.6.0, TiDB supports setting multiple PD addresses. | `tidb.pd-addr` |
```
Comment (hfxsd) [inline tidb-lightning/tidb-lightning-command-line-full.md:26]:
```suggestion
| `--pd-urls <host1:port1,host2:port2,...,hostn:portn>` | PD endpoint address. Starting from v7.6.0, TiDB supports setting multiple PD addresses. | `tidb.pd-addr` |
```

**Review by ran-huang — APPROVED** [review]

**Pull request merged**